swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -9,12 +9,15 @@
 import Foundation
 import FullStory
 
-/**
+/*
  FullStory Manager is a file full of functions and enums that can make the FullStory API easier to apply to areas of your project.
```

Comment:
Consider making this a bit more direct.  Instead of "FullStory Manager is a file full of functions and enums" perhaps say "FullStory Manager provides common functions and artifacts that make it easier to integrate FullStory into your iOS app." I think talking less about the design and more about how as a developer I can get maximum use and clarity from this is the goal.